.github/workflows/ci.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+
+on: [push]
+jobs:
+  markdown:
+    runs-on: ubuntu-latest
+    steps:    
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Run mdl        
+        uses: actionshub/markdownlint@1.2.0   
+        with:  
+          path: './docs'          
```

Comment:
Jeg tænker stadig at det ville være en fordel at have en newline i slutningen af filerne. Det var denne her (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) forklaring på hvorfor vi bruger newlines jeg læste og jeg tænker at selvom det i det store og hele ikke betyder så meget kan det vel aldrig være forkert at følge standarden. Og så kan det godt være at linteren ikke læser de her filer men jeg tænker s t a d i g at det er en god ide at følge standarden.